Taskfile.yml

```diff
@@ -98,3 +101,45 @@ tasks:
       - crowdin
     cmds:
       - crowdin upload sources
+
+  release:
+    desc: Create a release pull request
+    cmds:
+    - task: trigger-release-pull-request-workflow
+      vars: { WORKFLOW_FILE: 'release-pull-request.yml', RELEASE_TYPE: 'release'
+        }
+
+  hotfix:
+    desc: Create a hotfix pull request
+    cmds:
+    - task: trigger-release-pull-request-workflow
+      vars: { WORKFLOW_FILE: 'hotfix-pull-request.yml', RELEASE_TYPE: 'hotfix'}
+
+  trigger-release-pull-request-workflow:
+    internal: true
+    vars:
+      WORKFLOW_FILE: '{{default "create-release-pr.yml" .WORKFLOW_FILE}}'
+      RELEASE_TYPE: '{{default "release" .RELEASE_TYPE}}'
+    silent: true
+    preconditions:
+    - sh: gh --version
+      msg: |
+        ⚠️ This task requires `gh` (Github CLI).
+        Please check out: https://github.com/cli/cli#installation
+        You'll need a personal access token to authenticate with `gh`.
+        Please check out: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic
+        Then, run `gh auth login` or set the GITHUB_TOKEN environment variable with your personal access token.
+    cmds:
```

Comment:
This is kind of a distortion of what `task` is about. It is usually not recommended to write whole bash scripts in a Taskfile and should probably be a smarter bash/python script stored somewhere if it needs to be in that form.

But I guess it works :shrug: